### ISSUE TITLE
Add support for dotnet core 2.2 build

### ISSUE THREAD

**Issue by Andrioden** [GoogleCloudPlatform/cloud-builders#441]
I previously (#288) requested that you added support for 2.1, which you pleasantly did within 2 weeks, so here I am back asking for 2.2. This time I have not tested it live as I can se in [your documentation](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/dotnet) that it is not yet supported.

[NET Core 2.2 was released December 4th 2018](https://blogs.msdn.microsoft.com/dotnet/2018/12/04/announcing-net-core-2-2/)

**Comment by Andrioden** [comment]
@ILMTitan Sorry to tag you, but I believe it was you that solved the previous issue. Any thoughts on updating your build tools to dotnet core 2.2?

**Comment by ImJasonH** [comment]
There's an ongoing change being proposed to the dotnet builder in https://github.com/GoogleCloudPlatform/cloud-builders/pull/392

As part of that change, we're going to recommend users use the official dotnet image maintained by Microsoft and available on Dockerhub.

```
steps:
- name: microsoft/dotnet:2.2-sdk
  args: ['dotnet', 'build']
```

This has the benefit of being available today, and when `2.2.X` or `2.3` are released, those will be available immediately, you won't have to wait for us to update this dotnet builder.

Longer-term, we hope to stop relying on our custom images at all, since the official dotnet builders are better supported by real .Net experts at Microsoft.

**Comment by Andrioden** [comment]
Thanks for pointing me in the direction of the dotnet images. Upgraded my project from 2.1.2 -> 2.2.2 and it worked perfectly.

I compared the test run time (removing test execution time) to see if using an docker image for dotnet test execution had any negative impact. I compared it with the last 4 runs. It went from ~58sec to 45sec. So all around great.

Thanks again! Feel free to close the issue if you dont want to it to remind yourself about adding the "native" dotnet builders.
